ggshield/cmd/secret/ignore.py

```diff
@@ -7,41 +7,69 @@
 from ggshield.core.cache import Cache
 from ggshield.core.config import Config
 from ggshield.core.text_utils import pluralize
+from ggshield.core.types import IgnoredMatch
 
 
 @click.command()
+@click.argument(
+    "hash",
```

Comment:
We should call this `match` instead of `hash`, to be consistent with the way it's called in the `.gitguardian.yaml` file.

Reply:
Actually I changed my mind: in the text output of scans we use "Secret SHA". Since I expect the use case for this would be users running `ggshield secret scan ...`, getting a list of secrets and wanting to remove them, then it makes sense to use a similar name there.

Reply:
Makes sense, do you prefer me to put SHA or SECRET_SHA here?

Reply:
SECRET_SHA because it's less ambiguous.